src/agent0/ethpy/hyperdrive/interface/read_interface.py

```diff
@@ -109,6 +113,10 @@ def __init__(
             The number of times to retry the read call if it fails. Defaults to 5.
         txn_receipt_timeout: float | None, optional
             The timeout for waiting for a transaction receipt in seconds. Defaults to 120.
+        txn_options_base_fee_multiple: float | None, optional
```

Comment:
I'm not following how you're using these new arguments. It seems like they are passed to the individual write functions in addition to the interface constructor. I think it's better to pass them to the write functions and not be member attributes of the interface object, but I could be missing something.